cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesModel.java

```diff
@@ -109,7 +109,19 @@ default PropertyBags fullModels() {
 
     PropertyBags equivalentShunts();
 
-    PropertyBags nonlinearShuntCompensatorPoints(String id);
+    PropertyBags shuntCompensatorPoints();
+
+    // Non-linear shunt compensator points grouped by shunt compensator
+    Map<String, PropertyBags> groupedShuntCompensatorPoints();
```

Comment:
I don't think this method should be exposed in the API instead of the old `nonlinearShuntCompensatorPoints(String id)` method.
When we look at the usage, the new method is always called to retrieve the data for a single id and it is not trivial to refactor the code to call the method only once for all the IDs.
So I think this method is more an implementation detail of `CgmesModelTripleStore`.

Note that using the new method also introduces breaking changes that are not really needed.

Reply:
Providing this method makes the API of the CGMES model consistent with other cached data, like grouped transformer ends.

Reply:
I'll refactor the other methods to have a consistent behaviour:
Provide in the API (CgmesModel) the methods:
- xxx() that will query all data
- xxx(String linkedObjectId) that will query the data associated to the linkedObjectId
In the implementation (AbstractCgmesModel) the same methods where:
- xxx() runs the query
- xxx(String linkedObjectId) that will:
  - If run for the 1st time run the query, build a Map and cache it
  - Returns the data associated to the linkedObjectId from the cached Map

Reply:
Seen with @rcourtier:

- The `nonlinearShuntCompensatorPoints(String id)` was restored and `groupedShuntCompensatorPoints()` is no more exposed in the `CgmesModel` interface.
- A general refactoring of `CgmesModel`, providing more consistency, will be done in another PR.